src/ice.c

```diff
@@ -3989,28 +3990,31 @@ static gboolean janus_ice_outgoing_transport_wide_cc_feedback(gpointer user_data
 		while (g_hash_table_iter_next(&iter, NULL, &value)) {
 			janus_ice_peerconnection_medium *m = value;
 			if(m && m->type == JANUS_MEDIA_VIDEO && m->recv) {
-				medium = m;
-				break;
+				/* If a medium (or simulcast layer, if applicable) has not received data, its SSRC may be unknown. */
+				/* Pick the first valid SSRC we find across all considered mediums */
+				int i = 0;
+				for(i = 0; i < 3; i++) {
+					if(m->ssrc_peer[i] != 0)
+						break;
+				}
+
+				/* Stop if we found a valid SSRC/medium to use */
+				if(i < 3) {
+					ssrc_peer = m->ssrc_peer[i];
+					medium = m;
+					break;
+				}
```

Comment:
I think this could be simplified as

```
int i = 0;
for(i=0; i<3; i++) {
	if(m->ssrc_peer[i] > 0) {
		medium = m;
		ssrc_peer = m->ssrc_peer[i];
		break;
	}
}
if(medium && ssrc_peer)
	break;
```

as I feel the second check is a bit more "awkward" (and possibly error prone should we decide to increase the number of substreams in the future), and the whole block could be made more compact.